ingestion/setup.py

```diff
@@ -65,6 +65,10 @@ def get_long_description():
     },
 }
 
+pii_requirements = {
+    "en_core_web_sm@https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.5.0/en_core_web_sm-3.5.0.tar.gz#egg=en_core_web",
```

Comment:
could we add a comment on what are we installing here?

Reply:
Yes 👍